sda-commons-starter/build.gradle

```diff
@@ -0,0 +1,22 @@
+dependencies {
+  compile project(':sda-commons-server-auth')
+  compile project(':sda-commons-server-consumer')
+  compile project(':sda-commons-server-cors')
+  compile project(':sda-commons-server-dropwizard')
+  compile project(':sda-commons-server-jackson')
+  compile project(':sda-commons-server-prometheus')
+  compile project(':sda-commons-server-security')
+  compile project(':sda-commons-server-openapi:')
+  compile project(':sda-commons-server-trace')
+  compile project(':sda-commons-server-healthcheck')
+  compile project(':sda-commons-server-jaeger')
+  compile project(':sda-commons-server-opentracing')
+
+  testCompile project(':sda-commons-server-testing')
+  testCompile 'org.assertj:assertj-core'
```

Comment:
Is already imported by `sda-commons-server-testing`.

Reply:
I've removed the dependency..